### PR TITLE
Leader Board Project : Hit The API Milestone

### PR DESCRIPTION
**Pull request by CaptainBawa**
On this branch:

- [] I created a new game with the name of your choice by using the API
-  [] I implement a "Refresh" button that gets all scores for a game created by you from the API

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,8 +63,8 @@
 ### Key Features <a name="key-features"></a>
 
 - **[User_can_add_score]**
-- **[User_can_see_score_on_the_webpack]**
-- **[Scores_are_saved_in_local_storage]**
+- **[User_can_see_score_on_the_webpage]**
+- **[Scores_are_saved_and_displayed_from_api]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (Ismailco) [inline README.md:69]: - [ ] Kindly make sure to include a license in your readme file, as for now the LICENSE link redirect to a 404 page

File: src/modules/display.js [hunk 2]
```diff
@@ -1,14 +1,29 @@
...
+ The function sends a POST request to a specified API endpoint with user and score data and returns
+ the result.
+The user parameter is a string that represents the name or identifier of the user
+ whose score is being submitted.
+ The score parameter is a numerical value representing the score achieved by the
+ user in a game. It is used in the `displayScore` function to send a POST request to the server with
+ the user's score data.
+ The function `displayScore` is returning the `result` property of the JSON response from
+ the API.
+ */
```
Comment (Ismailco) [inline src/modules/display.js:13]: - [ ] [OPTIONAL] You don't need to describe the functionality of all the functions in your code

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @CaptainBawa, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AdnanOlarmmi — APPROVED** [review]
Hi @CaptainBawa,

This is Adnan Is-haq, your beloved code reviewer, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Approved office man

## Highlights
- Nice one fixing the required changes.
- Nice design.

## Optional suggestions
_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**